Google.PowerShell/PubSub/GcpsSubscription.cs

```diff
@@ -315,4 +314,139 @@ private void GetSubscriptions(IEnumerable<string> subscriptionNames)
             }
         }
     }
+
+    /// <summary>
+    /// <para type="synopsis">
+    /// Changes the config of a subscription.
+    /// </para>
+    /// <para type="description">
+    /// Changes the config of a subscription from push to pull and vice versa. The cmdlet can also be used to
```

Comment:
What happens if I set PullConfig twice on the same subscription? Does it throw or nothing? Might be worth noting.